bpy_speckle/ui/view3d.py

```diff
@@ -225,7 +224,6 @@ def draw(self, context):
                 subcol = row.column()
                 subcol.operator("speckle.send_stream_objects", text="Send")
                 subcol.prop(speckle, "send_script", text="")
-                area.prop(stream, "query", text="Filter")
```

Comment:
what's the purpsoe of removing this line?

Reply:
Ah, this is unrelated, I'm in the process of removing some dead code, and this was another bit of dead code.

Reply:
I think the intention here was to have a custom filter in on the graphql query for objects. But since we don't use a graphql request, this isn't really possible for us to implement. And its just been sitting here for at least 2 years